imports/plugins/core/accounts/client/containers/updatePassword.js

```diff
@@ -76,7 +75,12 @@ const wrapComponent = (Comp) => (
         } else {
           // Now that Meteor.users is verified, we should do the same with the Accounts collection
           Meteor.call("accounts/verifyAccount");
-          Router.go(this.props.onCompleteRoute);
+          const { storefrontUrls } = Reaction.getCurrentShop();
+          if (Reaction.hasAdminAccess()) {
+            Router.go("/operator");
+          } else {
+            window.location.href = `${storefrontUrls.storefrontHomeUrl}/signin`;
```

Comment:
Is it possible that `/signin` could be `/sign-in` or `/login` in a different storefront. Should there be another variable for `storefrontSignInUrl`?

Reply:
I had the same thought, I think it make sense to have that URL as one of the `storefrontUrls`. @kieckhafer & @aldeed what are y'all thoughts on adding the login URL?

Reply:
Yeah, I think that makes sense to add it, can't hurt.